bootstrapper/crates/bootstrapper/Cargo.toml

```diff
@@ -0,0 +1,69 @@
+[package]
+name = "madara-bootstrapper"
```

Comment:
not sure if the name should be `bootstrapper` or `madara-bootstrapper`

Reply:
I think that it would be more concise if we use `bootstrapper` now that we have all the pieces in the monorepo. Besides, orchestrator binary  is just `orchestrator`
cc: @HermanObst

Reply:
Yup, agree with @ftheirs